source/_patterns/02-organisms/components/hero-banner.yaml

```diff
@@ -0,0 +1,51 @@
+$schema: http://json-schema.org/draft-04/schema#
+type: object
+properties:
+  headingId:
+    type: string
+    minLength: 1
+  image:
+    allOf:
+      - $ref: ../../00-atoms/components/picture.yaml
+  subjects:
+    type: object
+    properties:
+      list:
+        type: array
+        minItems: 1
+        items:
+          type: object
+          properties:
+            name:
+              type: string
+              minLength: 1
+            url:
+              type: string
+              minLength: 1
+          required:
+            - name
+            - url
+    required:
+      - list
+  title:
+    type: string
+    minLength: 1
+  titleLength:
+    enum:
+    - small
+    - medium
+    - large
+  url:
+    type: string
+    minLength: 1
+  summary:
+    type: string
+    minLength: 1
+  authors:
+    type: string
+  meta:
+    $ref: ../../01-molecules/components/meta.yaml
+required:
+  - title
+  - url
+  - summary
```

Comment:
`image` can be required as we require it currently in covers. @chris-huggins do you envisage having hero banners without an image?

Reply:
@nlisgo No a hero banner should always have an image

Reply:
Image and meta are added as required fields